### ISSUE TITLE
About mask2 in cifp_margin

### ISSUE THREAD

**Issue by yiminglin-ai** [fuenwang/MixFairFace#4]
Hello, why in CIFP margin did you consider only `neg_logits` that are smaller than `pos_logits - self.m`? Why those larger`neg_logits`  should not be penalised? Thanks.

https://github.com/fuenwang/MixFairFace/blob/11691d330a3cde2e1c4eeb76211ec6d5b7752547/network.py#L29-L30

**Comment by fuenwang** [comment]
This part is modified from official CIFP code (https://github.com/Tencent/TFace/blob/99cc558fde0176f38ff051155b110bfd7f6cdb3a/recognition/torchkit/head/localfc/cifp.py#L12) 

I don't know the reason either. But I have tried many experiments on CIFP, and the implementation can really work and improve accuracy.

**Comment by yiminglin-ai** [comment]
Thanks Fu-En, your implementation is much more readable than the official code! Have you experimented without the `mask2`?

**Comment by fuenwang** [comment]
I guess so. Because I had removed the official code line-by-line to check the corresponding performance, and my network.py only keeps the necessary parts to improve performance.

**Comment by yiminglin-ai** [comment]
I see. My assumption is that the high `neg_logits` may be due to label errors, such as images with incorrect ID labels. Therefore, CIFP avoids using these `neg_logits` exclude these errors. They can be harmful especially towards the end of the training process.
